Test/Unit/Observer/OrderStatusObserverTest.php

```diff
@@ -57,7 +54,7 @@ public function testGivenOrderStateIsNewShouldReturnVoid(): void
         $order->method('getState')->willReturn('new');
         $orderStatusObserver = $this->createOrderStatusObserverObject();
         $this->event->method('getData')->willReturn($order);
-        $this->ordersEndpoint->expects($this->never())->method('sendStatus');
+        $this->paymentsEndpoint->expects($this->never())->method('addOrderStatusByMerchantOrderReference');
         $orderStatusObserver->execute($this->observer);
     }
 
```

Comment:
I don't see in your test the coverage if `addOrderStatusByMerchantOrderReference` or `getAlmaClient` return Exception. I suggest to add these test. WDYT ?

Reply:
I think it's a very good idea ! done !